dspace-api/src/main/resources/org/dspace/storage/rdbms/sqlmigration/oracle/V5.7_2017.04.11__DS-3563_Index_metadatavalue_resource_type_id_column.sql

```diff
@@ -19,5 +19,5 @@ begin
   exception
   when index_not_exists then null;
 end;
-
+/
```

Comment:
Note for the Oracle-impaired (like me):  this slash is needed to cause execution of the block above, because the block swallows the semicolon on line 21.  See https://stackoverflow.com/questions/1079949/when-do-i-need-to-use-a-semicolon-vs-a-slash-in-oracle-sql